tonytanaka/index.js

```diff
@@ -0,0 +1,16 @@
+// Question 1
+// NaN stands for Not-A-Number
+let ex1a = "hello" * 1;
+console.log(typeof ex1a);
```

Comment:
In line 4 you have to check if  ex1a is NaN or not! And result should be true or false!
console.log(isNaN(ex1a));

Reply:
ok I will incude it